.gitignore

```diff
@@ -36,4 +36,8 @@ tsconfig.
 yarn.lock
 migrations
 contract-addresses.json
-typechain-types
\ No newline at end of file
+typechain-types
+txstore.db
+manager/keystore
+workers/keystore
```

Comment:
I'm not sure we need these files here, because they're generated if and only if we set `workdir` with the current directory